public/docs/ts/latest/guide/forms.jade

```diff
@@ -1,83 +1,90 @@
 include ../_util-fns
 
 :marked
-  We’ve all used a form to log in, submit a help request, place an order, book a flight,
-  schedule a meeting, and perform countless other data entry tasks.
   Forms are the mainstay of business applications.
+  You use forms to log in, submit a help request, place an order, book a flight,
+  schedule a meeting, and perform countless other data-entry tasks.
 
-  Any seasoned web developer can slap together an HTML form with all the right tags.
-  It's more challenging to create a cohesive data entry experience that guides the
+  Any seasoned web developer can create an HTML form with all the right tags.
```

Comment:
Can we change the tone here to remove the "seasoned web developer" -> It is important to create a data entry experience that guides...